CONTRIBUTING.md

```diff
@@ -75,61 +140,29 @@ Example:
 def test_add_one_and_one_equals_two():
 ```
 
-### Adding a new command
-
-See class documentation on the [Command](src/code42cli/commands.py) class for an explanation of its constructor parameters.
-
-1. If you are creating a new top-level command, create a new instance of `Command` and add it to the list returned
-    by `load_commands()` function in `code42cli.main.MainSubcommandLoader`.
-
-2. If you are creating a new subcommand, find the top-level command that this will be a subcommand of in
-    `load_commands()` in `code42cli.main.MainSubcommandLoader` and navigate to its subcommand loader's `load_commands()` 
-    Then, add a new instance of `Command` to the list returned.
-
-3. For commands that actually are executed (rather than just being groups), you will add a `handler` function as a constructor parameter.
-   This will be the function that you want to execute when your command is run.
-   * _Positional_ arguments of the handler will automatically become _required_ cli arguments.
-     * The order that the positional arguments should be entered in on the cli is the same as the order in which they appear in the handler.
-   * _Keyword_ arguments of the handler will automatically become _optional_ cli arguments
-   * the cli argument name will be the same as the handler param name except with `_` replaced with `-`, and prefixed with `--` if optional.
-
-    For example, consider the following python function:
-
-    ```python
-    def handler_example(one, two, three=None, four=None):
-        pass
-    ```
-
-    When the above function is supplied as a `Command`'s `handler` parameter, the result will be a command that can be executed as follows
-    (assuming `cmd` is the name given to the command):
-
-    ```bash
-    $ code42 cmd oneval twoval --three threeval --four fourval
-    ```
+## Documentation
 
-4. To add descriptions to your cli arguments to appear in the help text, your command takes a function as the `arg_customizer` parameter.
-    The entire [`ArgConfigCollection`](src/code42cli/args.py) that was automatically created is supplied as the only argument to this function
-    and can be modified by it. See `code42cli.cmds.profile._load_profile_create_descriptions` for an example of this.
+Public functions, classes, and methods should have docstrings.
```

Comment:
Should this be more like `command functions should have docstrings` ?

Reply:
should actually be something like `command functions should have accompanying documentation`